src/Native/System.Native/pal_process.cpp

```diff
@@ -17,6 +17,12 @@
 #include <syslog.h>
 #include <unistd.h>
 
+#if HAVE_LIBPROC
+#include <libproc.h>
+#include <sys/proc_info.h>
+#include <sys/resource.h>
+#endif
```

Comment:
Thanks. :+1: 
I was feeling an urge to point it out again so it doesn't get forgotten. :smile: